doc/cephadm/host-management.rst

```diff
@@ -228,6 +228,83 @@ create a new CRUSH host located in the specified hierarchy.
 
 See also :ref:`crush_map_default_types`.
 
+OS Tuning Profiles
+==================
+
+Cephadm supports managing os tuning profiles that allow users to apply a set of
+sysctl settings to a given set of hosts. To do this, first create a yaml spec
+of the following format
+
```

Comment:
Suggest: 
Cephadm can manage and operating system tuning profiles that apply a set of
`sysctl` settings to sets of hosts. First, create a YAML spec file in the following format: